deploy/aws_ami/first-time-setup.sh

```diff
@@ -94,45 +78,25 @@ curl -s --location --request POST 'https://hook.integromat.com/dkwb6i52am93pi30o
 }' > /dev/null
 
 # Step 1: Download the templates
-echo "Downloading the configuration templates..."
-templates_dir="$(mktemp -d)"
-mkdir -p "$templates_dir"
+echo "Downloading the Docker Compose file..."
+mkdir -p "$install_dir"
 (
-    cd "$templates_dir"
-    curl --remote-name-all --silent --show-error \
-        https://raw.githubusercontent.com/appsmithorg/appsmith/master/deploy/template/docker-compose.yml.sh \
-        https://raw.githubusercontent.com/appsmithorg/appsmith/master/deploy/template/mongo-init.js.sh \
-        https://raw.githubusercontent.com/appsmithorg/appsmith/master/deploy/template/docker.env.sh \
-        https://raw.githubusercontent.com/appsmithorg/appsmith/master/deploy/template/nginx_app.conf.sh \
-        https://raw.githubusercontent.com/appsmithorg/appsmith/master/deploy/template/encryption.env.sh
+    cd "$install_dir"
+    curl -L https://github.com/appsmithorg/appsmith/raw/release/deploy/aws_ami/docker-compose.yml -o $PWD/docker-compose.yml
```

Comment:
Let's use `master` here instead of `release`.

```suggestion
    curl -L https://github.com/appsmithorg/appsmith/raw/master/deploy/aws_ami/docker-compose.yml -o $PWD/docker-compose.yml
```